src/web/wfs/src/test/java/org/geoserver/wfs/web/WFSLayerConfigTest.java

```diff
@@ -0,0 +1,42 @@
+/* (c) 2014 Open Source Geospatial Foundation - all rights reserved
+ * (c) 2001 - 2013 OpenPlans
```

Comment:
new file?

Reply:
So @jodygarnett the GeoTools part is merged but won't do anything until this one is also merged. Should we merge this one as is and fix later?

Reply:
The only reason I did not approve was the musical encoding help instructions